sql/core/src/test/resources/sql-tests/analyzer-results/ansi/array.sql.out

```diff
@@ -194,25 +194,8 @@ org.apache.spark.sql.catalyst.ExtendedAnalysisException
 -- !query
 select sort_array(array('b', 'd'), cast(NULL as boolean))
```

Comment:
I'm not sure if this belongs to `break changes`, because according to `sort_array's` definition (extends `NullIntolerant`), it's reasonable for it to return `NULL` in this scenario
https://github.com/apache/spark/blob/8023504e69fdd037dea002e961b960fd9fa662ba/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala#L1049-L1050

Reply:
from failing to working is not a breaking change.